### ISSUE TITLE
Left sidebar disappears when window size is adjusted from small to big.

### ISSUE THREAD

**Issue by Korokenichi** [Bl4Cc4t/GoodTwitter2#502]
**Please check the following before you continue**
- [ ] Bug happens with the newest version of GT2 (see the [changelog](https://github.com/Bl4Cc4t/GoodTwitter2/blob/master/doc/changelog.md))
- [ ] Checked [FAQ](https://github.com/Bl4Cc4t/GoodTwitter2#faq)
- [ ] Checked if the issue has not been reported before

**Describe the bug**
I don't seem to see any report about this but I've had this problem for a while.

The left sidebar apparently bugs out when I make the window smaller, flat out disappears. Wondering if this is just kinda unfixable at the moment.

**Screenshots**
![2022-11-25 20-30-48](https://user-images.githubusercontent.com/79259221/203987322-c9cf2b72-2a57-473b-b57c-7be4f902a1f2.gif)

**Your configuration**
 - Firefox 107.0, Chrome 107.0
 - Tampermonkey 4.18.1, Violentmonkey 2.13.3

**Comment by Bl4Cc4t** [comment]
this issue is known (I think it got mentioned in *some* issue in the past too.)

I won't fix it with this version of the script.
However, the rewrite I'm doing already has this fix included :)
I don't know when it is ready though.

Thanks for reporting.

**Comment by Korokenichi** [comment]
oooo there's a rewrite? sounds great.

**Comment by Bl4Cc4t** [comment]
Yep, it's made in typescript and should also have better support for older browsers & debugging thanks to babel & a lot more logging :)
